src/azul/chalice.py

```diff
@@ -116,6 +121,21 @@ def patched_event_source_handler(self_, event, context):
         if old_handler.__code__ != patched_event_source_handler.__code__:
             chalice.app.EventSourceHandler.__call__ = patched_event_source_handler
 
+    def _wrapping_middleware(self, event, get_response):
+        response = get_response(event)
+        if response.status_code >= 400:
+            parsed = parse_accept_header(event.headers.get('accept'))
+            text_html = parsed.find('text/html')
+            star_star = parsed.find('*/*')
+            if -1 < text_html and (star_star == -1 or text_html < star_star):
```

Comment:
I think it's more conventional to write the operands in this order. I certainly find it easier to read this way.
```suggestion
            if text_html > -1 and (star_star == -1 or text_html < star_star):
```